tests/unit/controlgroup/controlgroup_core.js

```diff
@@ -0,0 +1,89 @@
+module( "Controlgroup: Core" );
+
+function hasCornerClass( className, element ) {
+	if ( className ) {
+		return element.hasClass( className );
+	}
+
+	return element.attr( "class" ).match( /ui-corner/g );
+}
+
+test( "selectmenu: open/close corners", function() {
+	expect( 1 );
+	var element = $( ".controlgroup" ).controlgroup(),
+		selects = element.find( "select" ),
+		selectButton = selects.eq( 0 ).selectmenu( "widget" );
+
+		expect( 12 );
+
+		selects.eq( 0 ).selectmenu( "open" );
+
+		strictEqual( selectButton.hasClass( "ui-corner-tl" ), true,
```

Comment:
Lets group assertions with the relevant code. Here, drop the line break above, so that the open call and the assertion go together. Apply the same for the rest.